azure-cli/ad/Add-GroupToServicePrincipal.ps1

```diff
@@ -0,0 +1,33 @@
+function Add-GroupToServicePrincipal(
+    [Parameter(Mandatory = $true)]
+    [EnvironmentConfig] $environmentConfig,
+  
+    [Parameter(Mandatory = $true)]
+    [NamingConfig] $namingConfig,
+
+    [Parameter(Mandatory = $true)]
+    [string] $groupId,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $serviceInstance = ""
+) {  
```

Comment:
Fix indentation and break line between ) and {

Reply:
Done